lago/prefix.py

```diff
@@ -362,16 +369,14 @@ def _register_preallocated_ips(self, conf):
 
                 if nic['ip'] in net['mapping'].values():
                     conflict_list = [
-                        name
-                        for name, ip in net['mapping'].items()
+                        name for name, ip in net['mapping'].items()
                         if ip == net['ip']
                     ]
                     raise RuntimeError(
-                        'IP %s was to several domains: %s %s' % (
-                            nic['ip'],
-                            dom_name,
-                            ' '.join(conflict_list),
-                        ),
+                        'IP %s was to several domains: %s %s' %
+                        (nic['ip'],
+                         dom_name,
+                         ' '.join(conflict_list), ),
```

Comment:
This changes are not related to the commit, try not to mix indent/style changes with the commit changes please.